content/integrations/awsbeanstalk.md

```diff
@@ -15,6 +15,10 @@ AWS Elastic Beanstalk is an easy-to-use service for deploying and scaling web ap
 
 If you haven't already, set up the [Amazon Web Services integration first](/integrations/aws).
 
+Receiving Elastic Beanstalk metrics requires also to [enable the Enhanced Health Reporting](http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/health-enhanced.html) feature for your environment, and to configure your environment to [publish enhanced health metrics to CloudWatch](http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/health-enhanced-cloudwatch.html#health-enhanced-cloudwatch-console)
```

Comment:
To receive Elastic Beanstalk metrics, you must [enable the Enhanced Health Reporting](http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/health-enhanced.html) feature for your environment, and configure your environment to [publish enhanced health metrics to CloudWatch](http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/health-enhanced-cloudwatch.html#health-enhanced-cloudwatch-console).